src/cases/components/case-hearings-list/case-hearings-list.component.html

```diff
@@ -25,15 +25,25 @@
     </td>
     <td class="govuk-table__cell govuk-table-width-30-percent">
       <div *ngIf="hasUpdateAction" class="div-action">
-        <ng-container *ngIf="!hasAddEdit(hearing)"><a href="javascript:void(0)" (click)="viewAndEdit(hearing.hearingID)">View or edit</a>&nbsp;&nbsp;&nbsp;&nbsp;</ng-container>
-        <ng-container *ngIf="hasAddEdit(hearing)"><a href="javascript:void(0)" (click)="addAndEdit(hearing.hearingID)">Add or edit</a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</ng-container>
+        <ng-container *ngIf="!isAwaitingActual(hearing.exuiDisplayStatus) && !isNonCancellable(hearing.exuiDisplayStatus)">
+          <a id="link-view-or-edit-{{hearing.hearingID}}"
+             href="javascript:void(0)"
+             (click)="viewAndEdit(hearing.hearingID)">View or edit</a>&nbsp;&nbsp;&nbsp;&nbsp;
```

Comment:
What is the point of the non-breaking space?

Reply:
because the requirement asks for 4 spaces exiplicitely.